soccer/ui/MainWindow.ui

```diff
@@ -82,6 +82,9 @@
              <height>24</height>
```

Comment:
The font definitely seems to be monospace but it still seems to be jiggling. (And there's also the problem when you go from `99` -> `100`. This shouldn't be too hard to fix though, just needs a padding element (I think).